src/pages/overlay/components/AmbassadorList.tsx

```diff
@@ -11,10 +11,29 @@ import AnimalButton from '../../../utils/global/animalButton/AnimalButton'
 import styles from './ambassadorList.module.css'
 import arrow from '../../../assets/arrow.jpg'
 
+const SCROLL_OFFSET = 200;
```

Comment:
put variable inside scrollListToAnimal function. it is only used in that function.